projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.d.ts

```diff
@@ -1050,7 +1050,9 @@ interface JQuery {
   on(events: 'addrow', handler: JQuery.EventHandlerBase<any, SohoDataGridAddRowEvent>): this;
   on(events: 'click', handler: JQuery.EventHandlerBase<any, SohoDataGridRowClicked>): this;
   on(events: 'collapserow', handler: JQuery.EventHandlerBase<any, SohoDataGridRowCollapseEvent>): this;
+  on(events: 'expandrow', handler: JQuery.EventHandlerBase<any, SohoDataGridRowExpandEvent>): this;
   on(events: 'contextmenu' | 'dblclick', handler: JQuery.EventHandlerBase<any, SohoDataGridRowClicked>): this;
+  on(events: 'filtered', handler: JQuery.EventHandlerBase<any, SohoDataGridOpenFilteredEvent>): this;
```

Comment:
Would this be a breaking change?  Are we okay with that?

Reply:
Oh no I see, just change it to SohoDataGridFilteredEvent.

Reply:
I think it's ok to change this because I doubt many people rely on the type due to using <any> in soho-datagrid afterViewInit. A breaking change would be someone has used that type in their event handler. So maybe a entry into the breaking changes would be appropriate.

Reply:
Actually the component was correct, just the typings were wrong.  Have changed the typings so no need to a breaking change comment.